system-tables/system-table-information-schema.md

```diff
@@ -32,6 +32,16 @@ select * from `ANALYZE_STATUS`;
 6 rows in set
 ```
 
+`ANALYZE_STATUS` 表中列的含义如下：
+
+* `TABLE_SCHEMA`: 表所属的数据库的名称。
+* `TABLE_NAME`: 表的名称。
+* `PARTITION_NAME`: 分区表的名称。
+* `JOB_INFO`: 任务信息。
+* `PROCESSED_ROWS`: 进程数量。
```

Comment:
Are you sure?

Reply:
Not sure, it's a view that doesn't exist in MySQL.